pkg/operation/botanist/health_check.go

```diff
@@ -433,17 +419,31 @@ func (b *HealthChecker) CheckClusterNodes(
 }
 
 // CheckMonitoringSystemComponents checks whether the monitoring in the given listers are complete and healthy.
+// TODO: Enable below commented code and add a unit test after the next release.
 func (b *HealthChecker) CheckMonitoringSystemComponents(
 	namespace string,
 	isTestingShoot bool,
 	condition gardencorev1beta1.Condition,
+	// deploymentLister kutil.DeploymentLister,
 	daemonSetLister kutil.DaemonSetLister,
 ) (*gardencorev1beta1.Condition, error) {
 
 	if isTestingShoot {
 		return nil, nil
 	}
 
+	// deploymentList, err := deploymentLister.Deployments(namespace).List(monitoringSelector)
```

Comment:
Do we need this to be commented here?

Reply:
This is preparation for the health check for the blackbox exporter which wasn't implemented so far. Though, it's commented because we first have to change the role to `monitoring`, so we can only activate it with the next release.

Reply:
Got it. Although not necessary, a comment to state this would have been nice.

Reply:
There is comment, see https://github.com/gardener/gardener/pull/1830/files#diff-2b6dc6f4235e9f47c39f1f79d8d3e214R422